Gemfile.lock

```diff
@@ -483,7 +483,7 @@ GEM
       rspec-mocks (~> 3.5.0)
       rspec-support (~> 3.5.0)
     rspec-support (3.5.0)
-    rubocop (0.47.1)
+    rubocop (0.48.1)
```

Comment:
Looks like this introduced a bunch of new offenses. What do you think about checking for and correcting offenses every time Rubocop is updated? Since CC only runs Rubocop on changed files, it's not going to flag these new offenses, but when we run `make lint` locally, we'll see all the offenses. I like to be able to run `make lint` locally and see everything pass. It seems like most of the offenses are for using `%i` for an array of symbols, and those can be autocorrected with `rubocop -R -a`.